src/5.0_to_5.1/calculated-measures/getNamesOfCalculatedMeasuresToMigrate.test.ts

```diff
@@ -0,0 +1,14 @@
+import { contentServer } from "../__test_resources__/contentServer";
+import { getNamesOfCalculatedMeasuresToMigrate } from "./getNamesOfCalculatedMeasuresToMigrate";
+
+describe("getNamesOfCalculatedMeasuresToMigrate", () => {
+  it("returns the names corresponding to the calculated measure files created with ActiveUI 5.0", () => {
```

Comment:
I wanted to concey that it's only the saved ones, as opposed to query-scoped ones.
I'll do that by just adding "saved" with your wording, instead of my heavier one 👍